NoNicotineAPI/Program.cs

```diff
@@ -64,6 +67,7 @@
 
 builder.Services.AddSingleton<IAuthenticationService, AuthenticationService>();
 builder.Services.AddScoped<IPatientRepository, PatientRepository>();
+builder.Services.AddScoped<IEntryRepository, EntryRepository>();
```

Comment:
Shouldn't we also add a TherapistRepository?

Reply:
Yes